test/fixtures/rules/std_rule_cases/L036.yml

```diff
@@ -153,7 +151,59 @@ test_distinct_single_fail_b:
     SELECT
       distinct a
     FROM my_table
-  # NOTE: This doesn't yet reflow nicely. We should solve that elsewhere.
   fix_str: |
     SELECT distinct a
-       FROM my_table
+    FROM my_table
+
+test_long_single_select_clause:
+  fail_str: "SELECT\n100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000\n"
```

Comment:
The original issue happens with a combination of L016 and L36. Guess it doesn't really matter here since we're only running L036 so will remove this test case.

Reply:
I added a complex combo test case to capture this now, and removed it from these simple fixtures.